packages/index/index.1.3.0/opam

```diff
@@ -15,7 +15,7 @@ doc:          "https://mirage.github.io/index/"
 build: [
  ["dune" "subst"] {dev}
  ["dune" "build" "-p" name "-j" jobs]
- ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+ ["dune" "runtest" "-p" name "-j" jobs] {with-test & cmdliner:version < "1.1.0"}
```

Comment:
Why not adding:
```
"cmdliner" {with-test & < "1.1.0"}
```
to the dependencies instead?

Reply:
Good point!